client/web/web.go

```diff
@@ -218,23 +227,60 @@ func NewServer(opts ServerOpts) (s *Server, err error) {
 	return s, nil
 }
 
-func (s *Server) withCSRF(h http.Handler) http.Handler {
-	csrfProtect := csrf.Protect(s.csrfKey(), csrf.Secure(false))
+func (s *Server) CSRFProtect(h http.Handler) http.Handler {
```

Comment:
nit: since this doesn't need to be exported
```suggestion
func (s *Server) csrfProtect(h http.Handler) http.Handler {
```